### ISSUE TITLE
NOde.js problem still not solved

### ISSUE THREAD

**Issue by RobTillaart** [arduino/arduino-lint-action#414]
### Describe the problem

See #344 

### To reproduce

Ran this morning an Arduino-lint

### Expected behavior

NO error message or warning
now Im getting still this warning: 

- https://github.com/RobTillaart/INA228/actions/runs/11204451877/job/31157009073

Annotations
1 warning

The following actions use a deprecated Node.js version and will be forced to run on node20: arduino/arduino-lint-action@v1. For more info: https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/

### 'arduino/arduino-lint-action' version

arduino/arduino-lint-action@v4

### Additional context

#344

my yml file looks like

```
name: Arduino-lint

on: [push, pull_request]
jobs:
  lint:
    runs-on: ubuntu-latest
    timeout-minutes: 5
    steps:
      - uses: actions/checkout@v4
      - uses: arduino/arduino-lint-action@v1
        with:
          library-manager: update
          compliance: strict
```

which is functional identical to the one proposed in the readme.md file.

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-lint-action/issues?q=)
- [X] I verified the problem still occurs when using the latest version
- [X] My report contains all necessary details

**Comment by per1234** [comment]
@RobTillaart in fact it is solved, but you aren't using the version of the action that has the update.

When you do this:

```yaml
- uses: arduino/arduino-lint-action@v1
```
it causes the release version of the action to be used. There hasn't been a release of the action since Node.js was updated so it is expected that this will cause the version of the action configured for Node.js 16 to be used.

If you want to try the beta tester version of the action which is configured for Node.js, you must do this:

```yaml
- uses: arduino/arduino-lint-action@main
```

Or you can just wait for the next release of the action to come out (which should happen later today).
